<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferences</class>
 <widget class="QDialog" name="preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>541</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>541</width>
    <height>541</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>541</width>
    <height>541</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/Pictures/wallpaper.png</normaloff>:/icons/Pictures/wallpaper.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>401</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="toolTip">
       <string>Reset preferences to the default onces</string>
      </property>
      <property name="text">
       <string>Reset Preferences</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>&amp;Close</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="aplyButton">
      <property name="toolTip">
       <string>Save and close the dialog</string>
      </property>
      <property name="text">
       <string>&amp;Save</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>332</y>
     <width>521</width>
     <height>149</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Notification on Wallpaper change</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="playsoundcheckBox">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Play sound everytime a wallpaper changes</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="customsoundcheckBox">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Use custom sound</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="browsesoundButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Click to select your own sound</string>
      </property>
      <property name="text">
       <string>(None)</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icons/Pictures/audio-x-generic.png</normaloff>:/icons/Pictures/audio-x-generic.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Stop all possible sounds</string>
      </property>
      <property name="text">
       <string>Stop All Sounds</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>54</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Start Changing Desktop Wallpapers</string>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>190</y>
     <width>511</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Language: ( You will need to restart Wallch )</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>134</y>
     <width>511</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Default Style of wallpaper images:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Startup Options</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combostyle_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>511</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>Do not keep history</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Simple history (month and day of month)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Advanced history (month, day,hour,minute)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Very analytical history (month, day,hour,minute,second)</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="combostyle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>154</y>
     <width>511</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>Tile</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Zoom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Center</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Scale</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stretch</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Span</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>250</y>
     <width>511</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>History:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>34</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Change Desktop Wallpaper ( Once )</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>74</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Auto-enable Live Earth Wallpaper</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>511</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Notifications</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combostyle_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>511</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <item>
    <property name="text">
     <string notr="true">English (en)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Persian (fa)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>German (de)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">Greek (el)</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>105</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Minimize to tray ( Not recommened for Unity )</string>
   </property>
  </widget>
  <zorder>checkBox_3</zorder>
  <zorder>layoutWidget2</zorder>
  <zorder>layoutWidget4</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>label_22</zorder>
  <zorder>label_19</zorder>
  <zorder>checkBox_5</zorder>
  <zorder>combostyle_3</zorder>
  <zorder>combostyle</zorder>
  <zorder>label_23</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_3</zorder>
  <zorder>label_5</zorder>
  <zorder>combostyle_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
